lib.nbjshell9/build.xml

```diff
@@ -1,4 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
This header needs some newlines here. See some other XML files in the repository.

Better still, don't add a license header for build.xml

They are a "Problems to be solved centrally" (see https://cwiki.apache.org/confluence/display/NETBEANS/List+of+Modules+to+Review ) and we'll have a script update them all at once in the future.